config/rbac/rbac_role.yaml

```diff
@@ -0,0 +1,105 @@
+apiVersion: rbac.authorization.k8s.io/v1
```

Comment:
I think that we could try not to deviate too much from `kubebuilder`; we have to assess what real permissions we need for the controllers and use RBAC. I think it would be good to keep these rbac files too.